### ISSUE TITLE
Change email fields to not require confirmation (and any other confirmation fields)

### ISSUE THREAD

**Issue by joedolson** [wpa11yday/wpaccessibilityday#159]
As of WCAG 2.2, redundant entry of information violates SC 3.3.7. Our forms were built prior to WCAG 2.2 being published, but I don't think our usages meet any of the exceptions, and these should be switched. 

The first step will be auditing the forms to see what still requires these; some forms don't, but some still do (most notably, the registration form.)
